Stabilise unit tests for generateSeed

### Expected behaviour
All unit tests pass each time
### Actual behaviour
Sometimes this happes
```
FAILED TESTS:
  Passphrase
    generateSeed
      ✖ should generate a predictable sequence of bytes for given list of randoms
        Chrome 59.0.3071 (Mac OS X 10.12.6)
      AssertionError: expected [ Array(16) ] to deeply equal undefined
          at Assertion.assertEqual (src/utils/passphrase.test.js:62131:19)
          at Assertion.ctx.(anonymous function) [as equal] (src/utils/passphrase.test.js:65569:25)
          at src/utils/passphrase.test.js:77832:48
          at Array.forEach (<anonymous>)
          at Context.<anonymous> (src/utils/passphrase.test.js:77829:15)
```
### Steps to reproduce
Run unit tests several times

--- a/src/utils/passphrase.test.js
+++ b/src/utils/passphrase.test.js
@@ -95,7 +95,7 @@ describe('Passphrase', () => {
 
       let data;
       randoms.forEach((rand, i) => {
-        if (!data || data.percentage < 100) {
+        if ((!data || data.percentage < 100) && i < bytes.length) {
           data = generateSeed(data, rand);
           expect(data.byte).to.deep.equal(bytes[i]);
         }

